module_utils/fdm_swagger_client.py

```diff
@@ -188,7 +189,8 @@ def _get_operations(self, spec):
                     OperationField.METHOD: method,
                     OperationField.URL: self._base_path + url,
                     OperationField.MODEL_NAME: self._get_model_name(method, params),
-                    OperationField.RETURN_MULTIPLE_ITEMS: self._return_multiple_items(params)
+                    OperationField.RETURN_MULTIPLE_ITEMS: self._return_multiple_items(params),
+                    OperationField.TAGS: params.get(OperationField.TAGS, [])
```

Comment:
Please, update the unit tests for `fdm_swagger_client` to cover `tags`.

Reply:
done